.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run all crawlers
+      run: chmod +x docs-crawler/docs/run_crawlers.sh
+    - name: run aggregate_crawler
+      run: python docs-crawler/docs/aggregate_crawler_output.py
```

Comment:
Lastly, we want to do something here once the output has been generated. Either commit it to this branch, or fail if there are new changes. I'm okay with either but I think automatically committing the new changing might be more useful. What do you think?

Reply:
ok, that's great idea